artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionBridge.java

```diff
@@ -118,9 +119,20 @@ public ClusterConnectionBridge(final ClusterConnection clusterConnection,
                                   final MessageFlowRecord flowRecord,
                                   final TransportConfiguration connector,
                                   final String storeAndForwardPrefix,
-                                  final StorageManager storageManager) {
-      super(targetLocator, initialConnectAttempts, reconnectAttempts, 0, // reconnectAttemptsOnSameNode means nothing on the clustering bridge since we always try the same
-            retryInterval, retryMultiplier, maxRetryInterval, nodeUUID, name, queue, executor, filterString, forwardingAddress, scheduledExecutor, transformer, useDuplicateDetection, user, password, server, ComponentConfigurationRoutingType.valueOf(ActiveMQDefaultConfiguration.getDefaultBridgeRoutingType()));
+                                  final StorageManager storageManager) throws ActiveMQException {
+      super(targetLocator, new BridgeConfiguration()
+         .setInitialConnectAttempts(initialConnectAttempts)
+         .setReconnectAttempts(reconnectAttempts)
+         .setReconnectAttemptsOnSameNode(0) // reconnectAttemptsOnSameNode means nothing on the clustering bridge since we always try the same
+         .setRetryInterval(retryInterval)
+         .setRetryIntervalMultiplier(retryMultiplier)
+         .setMaxRetryInterval(maxRetryInterval)
+         .setFilterString(filterString.toString())
```

Comment:
There are numerous test failing now due to NPEs from this area. Presumably the older handling ofthe value in the parent dealt with a null while these new bits dont.